lib/pages/transfer-domain-precheck-page.js

```diff
@@ -6,6 +6,6 @@ import AsyncBaseContainer from '../async-base-container';
 
 export default class TransferDomainPrecheckPage extends AsyncBaseContainer {
 	constructor( driver ) {
-		super( driver, By.css( '.transfer-domain-step__precheck' ) );
+		super( driver, By.css( '.transfer-domain-step__precheck' ), null, 40000 );
```

Comment:
we try to avoid hardcoded values like this, maybe use `this.explicitWait * 2` which will give you the same result

Reply:
Great, thanks for letting me know, and for updating!